packages/combobox/src/Combobox.spec.tsx

```diff
@@ -1033,6 +1034,10 @@ describe('packages/combobox', () => {
           expect(highlight).toHaveTextContent('Banana');
         });
 
+        test.todo('Down arrow cycles highlight to top');
```

Comment:
should we add these in?

Reply:
I wanted to make minimal changes to Combobox in this PR. I'll add these in https://github.com/mongodb/leafygreen-ui/pull/1619